### ISSUE TITLE
V2: react: drag drop: very slow movement 

### ISSUE THREAD

**Issue by dmitrykurmanov** [surveyjs/survey-creator#1445]
most likely because of react state update

reproduces with big surveys

**Comment by dmitrykurmanov** [comment]
We should change "survey-elements" drag-drop implementation to avoid the problem. Instead of showing "ghost drag-drop element" as an HTML question it would be better to show it like in choices/matrices

**Comment by dmitrykurmanov** [comment]
With @tsv2013, we decided to investigate the problem further to understand why does React is so slow at 50 questions.

After the investigation, we've found and fix a little problem with keys but it doesn't increase performance a lot.

And then finally we've found a reason. It is **development mode**. [React has additional stuff in development mode](https://reactjs.org/docs/optimizing-performance.html#use-the-production-build). So we've tested it in **production mode** and the problem is gone XD
